docs/eventing/samples/parallel/mutual-exclusivity/README.md

```diff
@@ -153,8 +154,8 @@ kubectl create -f ./ping-source.yaml
 ### Inspecting the results
 
 You can now see the final output by inspecting the logs of the
-`me-event-display` pods. Note that since we set the `PingSource` to emit
-every minute, it might take some time for the events to show up in the logs.
+`me-event-display` pods. Note that since we set the `PingSource` to emit every
+minute, it might take some time for the events to show up in the logs.
```

Comment:
```suggestion
`me-event-display` pods.

**NOTE:** The `PingSource` to emit every minute, so it may take some time for the events to show up in the logs.
```